composer.json

```diff
@@ -19,10 +19,10 @@
         "symfony/framework-bundle": "~2.3",
         "symfony/assetic-bundle": "~2.1",
         "friendsofsymfony/rest-bundle": "~1.0",
-        "jms/serializer-bundle": "~0.12",
         "midgard/createphp": "~1.1"
     },
     "require-dev": {
+        "jms/serializer-bundle": "~0.12",
```

Comment:
it should then also be added to suggest .. would also be interesting to see if it also works with ``~1.0``